SignalServiceKit/src/Messages/Interactions/OWSContact.h

```diff
@@ -163,13 +164,16 @@ NSString *NSStringForContactAddressType(OWSContactAddressType value);
 
 #pragma mark - System Contact Conversion
 
-+ (nullable OWSContact *)contactForSystemContact:(CNContact *)systemContact;
-+ (nullable CNContact *)systemContactForContact:(OWSContact *)contact;
++ (nullable OWSContact *)contactForSystemContact:(CNContact *)systemContact
+                                     transaction:(YapDatabaseReadWriteTransaction *)transaction;
++ (nullable CNContact *)systemContactForContact:(OWSContact *)contact
+                                    transaction:(YapDatabaseReadTransaction *)transaction;
++ (nullable CNContact *)nameOnlySystemContactForContact:(OWSContact *)contact;
```

Comment:
It doesn't need to be public yet, but it will as soon as we pull `OWSContacts` out to a separate source file (and rename it to something like `OWSContactConversion`), which I'd like to do.